en/setup/dev_env_windows_cygwin.md

```diff
@@ -1,202 +1,225 @@
 # Windows Cygwin Toolchain
 
-## Ready to use MSI Installer Download
+This is the newest (and best!) toolchain for developing PX4 on Windows.
 
-Latest Download:
+The toolchain is portable, easy to install, and easy to use. 
+It supports build/upload of PX4 to NuttX targets (Pixhawk series controllers) and can run the JMAVSim/SITL simulator with significantly better performance than the other Windows toolchains. 
+
+
+<!--
+## Ready to use MSI Installer Download {#installation}
```

Comment:
@MaEtUgR I couldn't think of a reason to keep the legacy versions. So I removed the section and we go straight to installation (where the latest is included). Simplifies!